content/webinars/aws-anomaly-detection-guardduty-slack/index.md

```diff
@@ -0,0 +1,95 @@
+---
+# Name of the webinar.
+title: "Anomaly Detection with AWS GuardDuty and Slack"
+meta_desc: "In this episode, Lee Zen walks you through anomaly detection with Amazon GuardDuty + Slack using TypeScript and Pulumi."
+
+# A featured webinar will display first in the list.
+featured: false
+
+# If the video is pre-recorded or live.
+pre_recorded: true
+
+# If the video is part of the PulumiTV series. Setting this value to true will list the video in the "PulumiTV" section.
+pulumi_tv: true
+
+# The preview image will be shown on the list page.
+preview_image: "https://img.youtube.com/vi/PF7KnOB3fLk/hqdefault.jpg"
+
+# Webinars with unlisted as true will not be shown on the webinar list
+unlisted: false
+
+# Gated webinars will have a registration form and the user will need
+# to fill out the form before viewing.
+gated: false
+
+# The layout of the landing page.
+type: webinars
+
+# External webinars will link to an external page instead of a webinar
+# landing/registration page. If the webinar is external you will need
+# set the 'block_external_search_index' flag to true so Google does not index
+# the webinar page created.
+external: false
+block_external_search_index: false
+
+# The url slug for the webinar landing page. If this is an external
+# webinar, use the external URL as the value here.
+url_slug: "aws-anomaly-detection-guardduty-slack"
+
+# The content of the hero section.
+hero:
+    # The title text in the hero. This also serves as the pages H1.
+    title: "Anomaly Detection with AWS GuardDuty and Slack"
+    # The image the appears on the right hand side of the hero.
+    image: "/icons/containers.svg"
+
+# Content for the left hand side section of the page.
+main:
+    # Webinar title.
+    title: "Anomaly Detection with AWS GuardDuty and Slack"
+    # URL for embedding a URL for ungated webinars.
+    youtube_url: "https://www.youtube.com/embed/PF7KnOB3fLk"
+    # Sortable date. The datetime Hugo will use to sort the webinars in date order.
+    sortable_date: 2020-04-01T10:00:00-07:00
```

Comment:
Is this right?

Reply:
Yup, that is the date the video was uploaded to YouTube. This is not a new video but rather I transcribed an old video for SEO purposes.